### DISCUSSION TITLE
Can I deploy a custom client on hubs domain itself?

### DISCUSSION THREAD

**Discussion by Vraj1234** [mozilla/hubs#5611] [General Questions & Discussion] [answered]
Heyy, 
So I am working on building a custom client, the issue is, I don't want to invest in AWS or DigitalOcean yet, since its a very small scale project. I dont know how to get about it, as a developer, my only option is deploying it locally, but i cant have a collaborative environment in that.

So I need,
A collaborative environment,
A custom client, with custom code.

If anyone could help please let us know

**Comment by matthewbcool** [comment, marked as answer]
Thanks very much for your question.

The Hubs team has just released and is testing a closed beta of a hosted solution to make it easier to set up and organize your own Hubs. In this first version of our Hubs subscription service, you can choose a custom subdomain, but not a whole domain and currently there is no option to deploy custom clients. I imagine if there is enough interest, the Hubs team may be able to provide these services so please follow and support the solutions you would like to see in the community discord. More info on how to join Discord and request an invite to the beta if you are interested here: [Hubs Early Access announcement](https://twitter.com/MozillaHubs/status/1549462617034436608)
